matchms/Pipeline.py

```diff
@@ -326,7 +324,7 @@ def write_to_logfile(self, line):
             with open(self.logging_file, "a", encoding="utf-8") as f:
                 f.write(line + '\n')
 
-    def import_data(self, query_files, reference_files=None):
+    def import_spectra(self, query_files, reference_files=None):
```

Comment:
Somehow it became a matchms&co tradition to use "spectrums" in such cases. Since we also use it elsewhere in matchms like this I would change it to `import_spectrums`.